doc/webservices/extract_api.rst

```diff
@@ -0,0 +1,478 @@
+|                     |                                                                                    |
+|                     | **candidate** has an additional field ``amount_type``. Its value is always percent.|
+|                     |                                                                                    |
+|                     | **selected_values** is a list of candidates.                                       |
++---------------------+------------------------------------------------------------------------------------+
+| global_taxes_amount | **content** is a float                                                             |
++---------------------+------------------------------------------------------------------------------------+
+| invoice_id          | **content** is a string                                                            |
++---------------------+------------------------------------------------------------------------------------+
+| subtotal            | **content** is a float                                                             |
++---------------------+------------------------------------------------------------------------------------+
+| total               | **content** is a float                                                             |
++---------------------+------------------------------------------------------------------------------------+
+| supplier            | **content** is a string                                                            |
++---------------------+------------------------------------------------------------------------------------+
+
+``feature_result`` for the ``invoice_lines`` feature
+''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+It follows a more specific structure. It is basically a list of dictionaries where each dictionary represents an invoice line. Each value follows a :ref:`feature_result <webservices/extract_api/invoice_get_results/feature_result>` structure.
+
+.. container:: doc-aside
+
+    .. switcher::
+
+        .. code-block:: json
+
+            [
+                {
+                    "description": feature_result,
+                    "discount": feature_result,
+                    "product": feature_result,
+                    "quantity": feature_result,
+                    "subtotal": feature_result,
+                    "total": feature_result,
+                    "taxes": feature_result,
+                    "total": feature_result,
+                    "unit": feature_result,
+                    "unit_price": feature_result
+                },
+                ...
+            ]
+
+.. _webservices/extract_api/invoice_validate:
+
+/iap/invoice_extract/validate (POST)
+''''''''''''''''''''''''''''''''''''
+
+Description
+^^^^^^^^^^^
+Route that validates different features of an invoice. You can call it multiple times, but currently the system expects one call with all the features.
```

Comment:
Indeed it is not clear. The purpose of this route is to validate the result to improve our model. By calling validate you just send us a feedback and tell us if we where right or not. It doesn't have any impact on the invoice itself but it might have impact on the result of the future invoice the client will send us.

Reply:
Ah, so this is an explicit data sharing / confirmation thing.

Really cool, but yeah the description should be *way* clarified, I didn't understand the purpose at all.